game/static/game/css/auth.css

```
@@ -428,6 +532,17 @@ body.login-page .helptext {
         padding: 30px 20px;
     }
 }
+@keyframes cardEnter {
```

Stylelint (17.11.1)

[error] Expected keyframe name "cardEnter" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)